package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "dz02.09.24",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "npx mocha tests/example.test.js",
+    "greet": "echo 'Hello, world!'"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "devDependencies": {
+    "chai": "^5.1.1",
+    "lodash": "^4.17.21",
+    "mocha": "^10.7.3"
+  }
```

Comment:
Please install `chai` and `mocha` as dependencies, but `lodash` - as dev dependency